brave/brave-ios
# Fixes #765: Update shields off description

Closes https://github.com/brave/brave-ios/issues/765: Update shields off description

I saw this and reached out to Karen about updating the string to match desktop. Instead of dynamically changing the title, I added the snippet about shields being down.



## Submitter Checklist:



## Test Plan:

1. Turn shields off.
2. Make sure descriptions matches `Brave Shields are currently down for this site. You\'re browsing without any privacy and security protections.`

### Screenshots:

![image](https://user-images.githubusercontent.com/29072694/60137750-d18d6200-975c-11e9-9fb3-d9f493c4065b.png)


## Reviewer Checklist:
